### DISCUSSION TITLE
Call live-view mount(params, session, socket) with session-params

### DISCUSSION THREAD

**Discussion by Chrichton** [keathley/wallaby#665] [Q&A]
My mount function requires a session-parameter:
`
def mount(
        _params,
        %{
          "current_user_id" => current_user_id
        } = _session,
        socket
      )
`

How do I supply this session-parameter using Wallaby?

Thanks for your reply, Heiko

**Comment by mhanberg** [comment]
Generally during these style of browser based tests, you have your "user" log into your app the usual way by filling out and submitting the login form.

In doing so, any session (be it through the normal controller or a live view) should be present.
